### ISSUE TITLE
Add SVN access keys

### ISSUE THREAD

**Issue by pfefferle** [diso/wordpress-openid#54]
@willnorris can you add the same SVN secrets to the OpenID Repo, as you did in the XRDS-Simple? Then it is easier for me to update the plugin.

**Comment by pfefferle** [comment]
You are totally right! I thought I do not have access to the SVN repos, but I have.

**Comment by willnorris** [comment]
might as well go ahead and switch the xrds-simple plugin to your credentials as well 🤷🏻
